Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/Folder/GetIshFolder.cs

```diff
@@ -40,29 +40,35 @@ namespace Trisoft.ISHRemote.Cmdlets.Folder
     /// </summary>
     /// <example>
     /// <code>
-    /// $ishSession = New-IshSession -WsBaseUrl "https://example.com/InfoShareWS/" -PSCredential Admin
+    /// $ishSession = New-IshSession -WsBaseUrl "https://example.com/ISHWS/" -PSCredential Admin
     /// Get-IshFolder -FolderPath "\General\__ISHRemote\Add-IshPublicationOutput\Pub"
     /// </code>
     /// <para>New-IshSession will submit into SessionState, so it can be reused by this cmdlet. Returns the IshFolder object.</para>
     /// </example>
     /// <example>
     /// <code>
-    /// $ishSession = New-IshSession -WsBaseUrl "https://example.com/InfoShareWS/" -PSCredential Admin
+    /// $ishSession = New-IshSession -WsBaseUrl "https://example.com/ISHWS/" -PSCredential Admin
     /// (Get-IshFolder -BaseFolder Data).name
     /// </code>
     /// <para>New-IshSession will submit into SessionState, so it can be reused by this cmdlet. Returns the name of the root data folder, typically called 'General'.</para>
     /// </example>
     /// <example>
     /// <code>
     /// $ishSession = New-IshSession -WsBaseUrl "https://example.com/ISHWS/" -PSCredential "Admin"
-    /// $requestedMetadata = Set-IshMetadataFilterField -IshSession $ishSession -Name "FNAME" -Level "None"
+    /// $requestedMetadata = Set-IshMetadataFilterField -Name "FNAME" -Level "None"
     /// $folderId = 7598 # provide a real folder identifier
-    /// $ishFoldersRetrieve = Get-IshFolder -IshSession $ishSession -FolderIds @($folderId) -RequestedMetaData $requestedMetadata
-    /// $ishFields = $ishFoldersRetrieve.Folders[0].IshFields
-    /// $retrievedFolderName = $ishFields.GetFieldValue("FNAME", "None", "Value")
+    /// $ishFolder = Get-IshFolder -FolderId $folderId -RequestedMetaData $requestedMetadata
+    /// $retrievedFolderName = Get-IshMetadataField -IshField $ishFolder.IshField -Name "FNAME" -Level None -ValueType Value
```

Comment:
Don't use `.IshField` property... either use `-IshObject` instead of `-IshField` parameter or use `PSNoteProperty` so simply write `$ishFolder.name`